Sudo roadmap update (#16)
trifectatechfoundation/trifectatech-website: content/initiatives/workplan-sudo-rs.md

template = "initiatives/workplans/sudo-rs.html"
+++

### Current work

* Port sudo and su to FreeBSD
* Improve compatibility with the original sudo utility

> * Support for often-used configuration options in /etc/sudoers

Reviewer: ```suggestion
* Support for often-used configuration options in /etc/sudoers
* Ability to configure /etc/sudoers options on a per-user, per-command, per-host basis.
* Improved environment handling (SETENV)
```
This line item is definitely something we want to do because it's probably the #1 blocker for being a "drop in replacement". SETENV has slightly lesser priority but also something we should table.

This work is funded by [NLnet Foundation](https://nlnet.nl/project/sudo-rs/).

### Future Roadmap

**Improved Compatibility**

> * Ability to configure /etc/sudoers options on a per-user, per-command, per-host basis.
> * Improved environment handling (SETENV)

Reviewer: ```suggestion
```

> * Port sudo and su to BSD (MacOS, NetBSD)

Reviewer: ```suggestion
* Port sudo and su to other platforms (MacOS, NetBSD, ...)
```

Reviewer: Note, of course this leave the "improved compatibility" item a bit empty; but I'm confident new things will pop up to add here.
